var/spack/repos/builtin/packages/mapnik/package.py

```diff
@@ -36,7 +36,7 @@ class Mapnik(AutotoolsPackage):
     depends_on('sqlite+rtree', type=('build', 'link', 'run'))
     depends_on('libwebp')
 
-    conflicts('%gcc@9.0.0:')
+    conflicts('%gcc@9.0.0:9.0.2')
```

Comment:
Does GCC has a version 9.0.1 / 9.0.2? Didn't you mean 9.2?

Reply:
You are right, there is no 9.0.* anyway ; since modifying 'harfbuzz' to compile with 9.1.0, 9.2.0, 9.3.0, 10.2.0 and 5.4.0, 7.5.0
in fact I believe we could  simply remove this "conflicts('%gcc@9...)" line, as it becomes obsolete. Do you agree ?